cms/jinja2/templates/base.html

```diff
@@ -46,38 +46,49 @@
 
 {% set footer_menu = footer_menu | default(navigation_settings.footer_menu.localized) %}
 
+{% set badge_label = _("Beta") %}
```

Comment:
needs comments as to why it is done this way

Reply:
Added in [15c3c0c](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/15c3c0c2d12e189b21cd64ca74566b9eefba75d5)